src/eventhubs/azext_eventhub/custom.py

```diff
@@ -0,0 +1,82 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from azext_eventhub._utils import accessrights_converter
+
+from azext_eventhub.eventhub.models import (EHNamespace, Sku, Eventhub, CaptureDescription, Destination)
+
+
+# Namespace Region
+def cli_namespace_create(client, resource_group_name, namespace_name, location, tags=None, sku='Standard', skutier=None, capacity=None, is_auto_inflate_enabled=None, maximum_throughput_units=None):
+    return client.create_or_update(resource_group_name, namespace_name, EHNamespace(location, tags,
+                                                                                    Sku(sku,
+                                                                                        skutier,
+                                                                                        capacity), is_auto_inflate_enabled, maximum_throughput_units))
+
+
+def cli_namespace_list(client, resource_group_name=None, namespace_name=None):
```

Comment:
You're not using namespace_name in this method so it should be removed.

Reply:
Also, you can then just have:
```
if resource_group_name:
    return client.list_by_resource_group(...)
return client.list()
```

Reply:
Resolved